Symfony/CS/Tests/Tokenizer/TokenTest.php

```diff
@@ -240,4 +240,101 @@ public function testPropertiesOfNonArrayToken()
         $this->assertNull($token->getLine());
         $this->assertFalse($token->isArray());
     }
+
+    /**
+     * @dataProvider provideEquals
+     */
+    public function testEquals($token, $equals, $other, $caseSensitive = null)
+    {
+        if (!isset($caseSensitive)) {
```

Comment:
* The isset is an old habit. Will fix across the board.
* The code was meant to test the method parameter default. If someone the signature somehow changes, the test would still pass.
* I do agree one test to do the above should be enough though